workspaces/liferay-partner-workspace/client-extensions/liferay-partner-etc-spring-boot/src/main/java/com/liferay/partner/QueueListener.java

```diff
@@ -140,6 +302,40 @@ HttpHeaders.AUTHORIZATION, _getAuthorization()
 			).block());
 	}
 
+	private long _getAccountRolesId(
+		String accountExternalReferenceCode, String accountRoleName) {
+
+		JSONObject accountRolesResponseJSONObject = _get(
+			uriBuilder -> uriBuilder.path(
+				StringBundler.concat(
+					"/o/headless-admin-user/v1.0/accounts",
+					"/by-external-reference-code/",
+					accountExternalReferenceCode, "/account-roles")
+			).queryParam(
+				"pageSize", "-1"
+			).build());
+
+		if (accountRolesResponseJSONObject.getInt("totalCount") <= 0) {
```

Comment:
do we need this? I think items is always there and will be empty if there are no results

Reply:
true, we don't need it